contents/blog/alerts-examples.mdx

```diff
@@ -0,0 +1,193 @@
+
+<ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/posthog_alert_df58e89828.png"
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/posthog_alert_df58e89828.png"
+    alt="Pricing pageviews alert"
+    classes="rounded"
+/>
+
+### 2. Get notified for errors and 404s
+
+You can also use alerts to track down errors. For example, you can track the number of visits to your 404 page. If someone visits `posthog.com/some-nonexistent-url`, they will land on our 404 page. Some number of these kinds of visits to this page is to be expected. Anyone that misspells a URL or finds a dead link is bound to land here at some point.
+
+But what about anomalous spikes in 404s? It might mean you accidentally deployed a broken link. Or a page has been moved and you need to set up a redirect to prevent users from missing it.
+
+In this case, you can set up another trend insight to graph the number of pageviews for your 404 page. You can examine the graph to establish a baseline for a standard amount of traffic to the 404 page.
+
+<ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/404_graph_light_089709bc1e.png"
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/404_graph_79107f52bb.png"
+    alt="404 graph"
+    classes="rounded"
+/>
+
+Now, we can set up an alert to check when 404s are significantly higher than this baseline.
+
+<ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/404_alert_light_c16295d3e4.png"
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/404_alert_dark_a014c1aa46.png"
+    alt="404 alert"
+    classes="rounded"
+/>
+
+### 3. Alert if important emails are not delivered
+
+Alerts can also be used to check for the _absence_ of something happening.
+
+For example, PostHog tracks the number of transactional billing emails that get sent out every day. If these emails aren't getting sent, it would be a big deal.
+
+![Ignoring warnings](https://frinkiac.com/video/S08E23/XOUcScbq8W_7yoA6nti16cCC31A=.gif)
+
+To do something similar, you can track events when you trigger transactional emails. Then, create an insight to track the number of emails sent every day.
+
+Then, we can set up an alert called "No billing events sent" that gets triggered when the transactional emails have a value less than 1. In other words, it will trigger the alert if no emails get sent.
+
+<ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/transactional_email_alert_light_57017a6227.png"
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/transactional_email_alert_dark_1fda09248e.png"
+    alt="Billing email alert"
+    classes="rounded"
+/>
+
+We can set the frequency to check this daily. And for critical alerts like this, you can add more than one email to the notification list and ensure the entire billing team receives the alert.
+
+### 4. Alert on changes in conversion rates
```

Comment:
Instead of having two separate lines could we use formula mode here? Might be more realistic.